gitlab-pipeline/stage/trigger-packages.yml

```
@@ -0,0 +1,29 @@
+trigger:mender-dist-packages:
+  stage: trigger-packages
+  needs: []
+  rules:
+    - if: $BUILD_MENDER_DIST_PACKAGES == "true"
+    - if: $PUBLISH_RELEASE_AUTOMATIC == "true"
+  variables:
+    # Unset DOCKER_ variables, the pipeline will execute on shared runners
+    DOCKER_HOST: "tcp://docker:2375"
+    DOCKER_CERT_PATH: ""
+    DOCKER_TLS_VERIFY: ""
+    DOCKER_TLS_CERTDIR: ""
+
+    # Mender release tagged versions:
+    MENDER_VERSION: $MENDER_REV
+    MENDER_CONNECT_VERSION: $MENDER_CONNECT_REV
+    MENDER_MONITOR_VERSION: $MENDER_MONITOR_REV
```

Note to self, this should be:
```suggestion
    MENDER_MONITOR_VERSION: $MONITOR_CLIENT_REV
```